pom.xml

```diff
@@ -45,10 +45,12 @@ under the License.
     <modules>
         <module>flink-connector-pulsar</module>
         <module>flink-sql-connector-pulsar</module>
+        <module>flink-connector-pulsar-e2e-tests</module>
     </modules>
 
     <properties>
         <flink.version>1.16.0</flink.version>
+        <pulsar.version>2.10.1</pulsar.version>
```

Comment:
should this be de-duplicated with the version in `flink-connector-pulsar`?

Reply:
Yes missed it, fixed now